src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -323,34 +317,36 @@ static void uct_rdmacm_cm_ep_destroy_dummy_qp(uct_rdmacm_cm_ep_t *cep)
     }
 
     cep->qp = NULL;
+    --ctx->num_dummy_qps;
 }
 
-static ucs_status_t uct_rdmacm_cm_create_dummy_qp(struct rdma_cm_id *id,
-                                                  struct ibv_cq *cq,
-                                                  struct ibv_qp **qp_p)
+static ucs_status_t
+uct_rdmacm_cm_create_dummy_qp(uct_rdmacm_cm_ep_t *cep,
+                              uct_rdmacm_cm_device_context_t *ctx)
```

Comment:
swap args order to match uct_rdmacm_cm_ep_destroy_dummy_qp

Reply:
done